tools/integration_tests/util/setup/setup.go

```diff
@@ -394,3 +394,61 @@ func AreBothMountedDirectoryAndTestBucketFlagsSet() bool {
 	log.Print("Not running mounted directory tests as both --mountedDirectory and --testBucket flags are not set.")
 	return false
 }
+
+func separateBucketAndObjectName(bucket, object *string) {
```

Comment:
Is it possible to split this PR into two PRs? - (a) Setup refactoring PR (b) List empty managed folders